owl/services/web_search/bing_search_service.py

```diff
@@ -0,0 +1,15 @@
+import httpx
+from ...models.bing import BingSearchResponse
+
+class BingSearchService:
+    def __init__(self, config):
```

Comment:
Can you add a type annotation here to clarify that config is actually BingSearchConfiguration not Configuration?

Reply:
Done